### DISCUSSION TITLE
Pages source files fetched from wrong location

### DISCUSSION THREAD

**Discussion by Ka-Repo** [community/community#23849] [Other Feature Feedback, Questions, & Ideas]
<p>Hey guys,</p>
<p>when I serve my website <strong>without</strong> Custom Domain everything works fine. When I add a custom domain the network requests for the source files go to that website, while the files are still hostet on github? The page remains empty.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/4/0460d6379b5ebe9ec896e92004fb7215ead5944f.png" data-download-href="/uploads/short-url/CJnFp4P5tUcOhjmdyO26WPpgL5.png?dl=1" title="2021-09-27 11_09_54-Window" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/29923510/181079696-8ffca6b2-1348-4504-9d95-0ea82435ac19.png" alt="2021-09-27 11_09_54-Window" data-base62-sha1="CJnFp4P5tUcOhjmdyO26WPpgL5" width="690" height="143" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/4/0460d6379b5ebe9ec896e92004fb7215ead5944f_2_690x143.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/4/0460d6379b5ebe9ec896e92004fb7215ead5944f_2_1035x214.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/0/4/0460d6379b5ebe9ec896e92004fb7215ead5944f.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/4/0460d6379b5ebe9ec896e92004fb7215ead5944f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2021-09-27 11_09_54-Window</span><span class="informations">1343×279 19.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/d/2d02a489aeef17ff6b8a1320b4b26f76bd15e08e.png" data-download-href="/uploads/short-url/6qb9bCsVkm6lFOfcSShKoZ8HW4C.png?dl=1" title="2021-09-27 11_08_46-Window" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/29923510/181079698-54a0d7b5-28fb-44ed-93fe-d674fdc68c73.png" alt="2021-09-27 11_08_46-Window" data-base62-sha1="6qb9bCsVkm6lFOfcSShKoZ8HW4C" width="690" height="361" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/d/2d02a489aeef17ff6b8a1320b4b26f76bd15e08e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2021-09-27 11_08_46-Window</span><span class="informations">941×493 24.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>CNAME file contains the custom domain name. DNS points to github ip adress (185.199.108.153) using A Record. My hoster only provides the option of setting 1 A-Record, could this be a problem or are multiple A-Records just used as fallback?</p>
<p>Appreciate the help :slight_smile:</p>

**Comment by yoannchaudet** [comment]
<p>Hello <a class="mention" href="/ka-repo">@Ka-Repo</a> and welcome here,</p>
<p>Setting a custom domain on your Pages site usually also changes the routing logic for your site.</p>
<p>In your specific example you went from <code>https://ka-repo.github.io/metaplex</code> to <code>https://example.com</code>.</p>
<p>You will want to update your site so resources are fetched at the root (<code>/</code>) of your site instead of under <code>/metaplex</code>.</p>
<p>I hope it helps,</p>

**Comment by Ka-Repo** [comment]
<p>Hi <a class="mention" href="/yoannchaudet">@yoannchaudet</a>,<br>
thanks for the reply!</p>
<p>EDIT: You saved my day. Had some prefix in my deploy command in my package.json.</p>
